src/Formik.tsx

```diff
@@ -314,10 +314,8 @@ export function useFormik<Values = object>({
         activeElement instanceof HTMLButtonElement
       ) {
         warning(
-          !!(
```

Comment:
If it was `!!`, shouldn't it be `!` now? It seems that this will start triggering false positives.

Reply:
nice catch, thanks

Reply:
The test fails if you make this change